main.py

```diff
@@ -237,6 +262,13 @@ def check_storehouse_consistency(self, switch):
                         f"Consistency check: alien flow on switch {dpid}, dpid"
                         " not indexed"
                     )
+                    archived_flows.append(
```

Comment:
There's still a risk of losing this `append` here if `_install_flows` side effect below crashes, fixing #26 that's on our radar would mitigate this risk.